### PR TITLE
Update status.js

### PR DESCRIPTION
**Pull request by sbharga**

### CODE REVIEW

**Comment by sbharga** [comment]
do it

File: status.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ function status(){
     "People memeing BIDOME",
     "!help in {servers} servers",
     "!status",
-    "2020 come to a close"
+    "2020 come to a close",
+    "blocks_n_more nerd"
```
Comment (Bloxs) [inline status.js:9]: why, just why

Comment (sbharga) [reply status.js:9]: well I'm not wrong

Comment (Bloxs) [reply status.js:9]: Smh my head

**Pull request closed without merging**